modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -760,6 +780,16 @@ namespace cv {
                         tensor_shape[1] = 1;
                         tensor_shape[2] = 1;
                     }
+                    else if (layer_type == "crop")
+                    {
+                        int crop_height = getParam<int>(layer_params, "crop_height", 3);
+                        int crop_width = getParam<int>(layer_params, "crop_width", 2);
```

Comment:
Let's avoid using default values here - the arguments are required.

Reply:
@dkurt Yes sir will correct them.

Reply:
What can I use instead of default values? The function demands initial values.

Reply:
Use zero values and check that after retrieving they are not zero (means values are presented)